posthog-web/src/posthog-web.ts

```diff
@@ -50,6 +56,9 @@ export class PostHog extends PostHogCore {
   }
 
   fetch(url: string, options: PostHogFetchOptions): Promise<PostHogFetchResponse> {
+    // TODO: what to do here?
+    // should we move this to core? https://github.com/PostHog/posthog-js-lite/blob/main/posthog-node/src/fetch.ts
+    // and reuse it here? if window isn't available?
```

Comment:
I don't really get what this means. How can we fetch if there is no window?

Reply:
In both Service Workers and Web Workers, there is no Window object, but the fetch() function is still available globally as part of the worker environment.
So I was assuming that https://github.com/PostHog/posthog-js-lite/blob/main/posthog-node/src/fetch.ts does something similar and we could reuse it here as a fallback if `window` isn't available.
Does that make sense?

Reply:
Ah i see. Yeah then I guess that [same line](https://github.com/PostHog/posthog-js-lite/blob/main/posthog-node/src/fetch.ts#L14-L17) would I think work for loading fetch from the global instead of window? 

Might be a bit tricky to test...

Reply:
yep, that looks like the fix to me...